packages/core/src/app/payment/PaymentForm.spec.tsx

```diff
@@ -247,12 +248,14 @@ describe('PaymentForm', () => {
     it('does not pass form values to parent component if validation fails', async () => {
         const handleSubmit = jest.fn();
         const container = mount(<PaymentFormTest
-            { ...defaultProps }
-            onSubmit={ handleSubmit }
-            validationSchema={ getCreditCardValidationSchema({
+            {...defaultProps}
+            onSubmit={handleSubmit}
+            // TODO: need to fix type for this schema
+            // @ts-ignore 
+            validationSchema={getCreditCardValidationSchema({
```

Comment:
```
Type 'ObjectSchema<CreditCardFieldsetValues>' is not assignable to type 'ObjectSchema<Partial<PaymentFormValues>>'.
```

Validation schema doesn't match, I was curious what we should do in this situation

Reply:
Will use `as` keyword to fix this